### ISSUE TITLE
When element having aria-label and title it reads both content.

### ISSUE THREAD

**Issue by kmkrish001** [FreedomScientific/VFO-standards-support#697]
### Summary

> When element having aria-label and title it reads both content.

### Expected result

> Need to read the aria-label only 

### Actual result

> Reads both aria-label and title.

### Example

> https://stackblitz.com/edit/typescript-laaqww?file=index.html

use tab key to focus button, then will see that both aria-label and title content was read.

#### JAWS version and build number

JAWS 2022
 
#### Operating System and version

Windows 10

#### Browser and version:

chrome 109.0.5414.120

**Comment by stevefaulkner** [comment]
@kmkrish001 it is he expected and correct behaviour for the aria-label and the title attribute content to be announced.
When a <button> has both the aria-label= the accessible name and the title= accessible description' both of which are announced by default for controls. Refer to [4.1.4 button Element Accessible Name Computation](https://www.w3.org/TR/html-aam-1.0/#button-element-accessible-name-computation) and [4.2 Accessible Description Computation](https://www.w3.org/TR/html-aam-1.0/#accdesc-computation)

**Comment by JAWS-test** [comment]
in your example the value of aria-label is "Remove button, press enter to activate", the visual label is "click"

Please note the following things:
* the aria-label should not name the element type (button), because this is already automatically output correctly by AT like screen readers, as long as proper HTML elements (`<button>`) or aria roles (`role=button`) are used  
* the aria-label should not contain any operating instructions, because these are automatically derived from the role. If hints are necessary because the operation differs from the standard, they should be included in the description (e.g. via title).
* the visual label must be included in the aria-label or must match the aria-label (https://www.w3.org/WAI/WCAG22/Understanding/label-in-name.html).

**Comment by rr-it** [comment]
The following does not work with NVDA.

For web-developers:

~If the `title`-attribute is somehow necessary, but should not be announced, then just set an empty `aria-description=""`.~

```html
<button
  aria-label="Some action"
  aria-description=""
  title="This is ignored as aria-description is set."
>This is ignored as aria-label is set.</button>
```

~Readout: " 'Some action' button"~

Also see:
https://w3c.github.io/aria/#aria-description
https://html.spec.whatwg.org/multipage/dom.html#attr-title

----

Using a null character hack actually works with NVDA:

```html
<button
  aria-label="Some action"
  aria-description="&#0;"
  title="This is ignored as aria-description is set."
>This is ignored as aria-label is set.</button>
```
